audit/projects/k8s-staging-test-infra/services/logging/logs.json

```diff
@@ -1,3 +1,4 @@
 [
-  "projects/k8s-staging-test-infra/logs/cloudaudit.googleapis.com%2Factivity"
+  "projects/k8s-staging-test-infra/logs/cloudaudit.googleapis.com%2Factivity",
+  "projects/k8s-staging-test-infra/logs/cloudbuild"
```

Comment:
I manually triggered a build in this project, ref: https://github.com/kubernetes/test-infra/pull/22450#issuecomment-858747549